Sources/Mantis/CropViewController/CropViewController.swift

```diff
@@ -62,8 +62,9 @@ public class CropViewController: UIViewController {
     private var orientation: UIInterfaceOrientation = .unknown
     private lazy var cropView = CropView(image: image, viewModel: CropViewModel())
     private var cropToolbar: CropToolbarProtocol
-    private var ratioPresenter: RatioPresenter?
+    private var ratioSelector: RatioSelector!
```

Comment:
I think we should keep ratioPresenter. If you want to add ratioSelector, you can make it an option by using Mantis.Config.

Reply:
should we hide ratioSelector or ratioPresenter when user select one of them

Reply:
I think we should only show one of them and hide the other one.